Source/WTF/wtf/Assertions.h

```diff
@@ -585,19 +585,37 @@ static constexpr bool unreachableForValue = false;
 
 #if ERROR_DISABLED
 #define LOG_ERROR(...) ((void)0)
+#define LOG_ONCE(SiteIsolation, ...) ((void)0)
```

Comment:
channel, not SiteIsolation.  Or it could just be ...

Reply:
Whoops, that one fixed locally dunno how the upload missed it.